utilities/adapter-registry.js

```diff
@@ -33,7 +33,7 @@ module.exports = CoreObject.extend({
   },
 
   _mergeDeployAdapters: function(addon) {
-    if (addon.type === 'ember-deploy-addon') {
+    if (addon.type === 'ember-cli-deploy-addon') {
```

Comment:
Yes, that would make sense to wait to change this I think. Most plugins should just work with holding off to make this change unless the plugin depended on `ember-deploy` directly.

Reverted d897962